### ISSUE TITLE
Lunacy Flatpack

### ISSUE THREAD

**Issue by TheBugurtov** [flathub/com.icons8.Lunacy#37]
Hi Garret!
I'm one of the developers behind Lunacy, the design software. I noticed that you're supporting an unofficial Flatpack version of Lunacy, and I wanted to talk about it.
Our team has decided to create an official Flatpack for Lunacy. However, we ran into a small issue – the name "Lunacy" is already taken for a Flatpack.
I'm reaching out to kindly ask if you would be willing to transfer the name "Lunacy" to us for our official release. This way, we can ensure that users easily find and trust the official distribution from our development team.
We're excited about collaborating with you and can help with any necessary steps to make the transfer smooth. Our aim is to provide users with a seamless experience and avoid any confusion.
If you're open to this idea, please let me know.
Best regards,
Alex
avbestix@gmail.com

**Comment by Beryesa** [comment]
@jacokok @garrett @barthalion ping!
Can you consider a takeover?

**Comment by Beryesa** [comment]
@TheBugurtov off-topic, be careful with flat**pak**, it's not **pack** !! :P

**Comment by garrett** [comment]
Oh, sorry; I didn't see this until just now (I was on PTO rather recently and probably accidentally cleared the notification). Thanks for pinging me again!

Yes, I'd be happy if you took this over! :tada:

**Comment by speakradmin** [comment]
Lunacy is a great piece of software. It would be fantastic to have an official flatpak available!

**Comment by garrett** [comment]
Hi! I'd still love to hand this over to someone official.

(I need somehow know the person is actually officially part of Lunacy before doing so.)

**Comment by TheBugurtov** [comment]
@garrett 

Hi, Garrett!

Our developer has mentioned that he attempted to reach out to you via email several times, but unfortunately, has not received any response. It's possible that his emails ended up in your spam folder, or there might be an issue with the email address used. Could you please check and confirm? We apologize for having to communicate through this platform, but it seems to be the only available means of reaching you at the moment.

**Comment by barthalion** [comment]
@TheBugurtov Hmm, not sure what happened there. Mind shooting me an e-mail at `b@bpiotrowski.pl`?

**Comment by TheBugurtov** [comment]
@barthalion Please check - you should have received an email from lunacy@icons8.com.

**Comment by barthalion** [comment]
@TheBugurtov should be done now. Once you accept the invite, please log it at https://flathub.org/login and follow the steps to verify Lunacy. Cheers!

**Comment by MichaelTunnell** [comment]
I see this is now official which is great but the description of the flatpak still says "This wrapper is not verified by, affiliated with, or supported by Icons8 LLC."

**Comment by garrett** [comment]
> Our developer has mentioned that he attempted to reach out to you via email several times, but unfortunately, has not received any response.

I had someone contact me _once_ via email several months ago from someone who didn't have a Lunacy related email address and whose GitHub account was mostly empty. I replied back, asking for some way to verify who he was and I never got a response.

I did search my spam folder a few times and everything. Sometimes people make a typo on my name (even in this thread), leaving out a letter. Perhaps it was that?

> now official which is great

Hooray! I'm happy it's official now. :partying_face: Congrats!
